### PR TITLE
Added postgres notes

### PR DESCRIPTION
**Pull request by lpatmo**
See: #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,20 +7,44 @@ The API spec all the proof-of-concepts: https://app.swaggerhub.com/apis-docs/bil
...
+- Fork this repo
+- Make sure you have Python 3 and Postgres 11.1 installed
+
+---
+
+You can either set up Postgres locally or using Docker.
+
+### Set up Postgres with Docker:
+
+[https://hackernoon.com/dont-install-postgres-docker-pull-postgres-bee20e200198](https://hackernoon.com/dont-install-postgres-docker-pull-postgres-bee20e200198)
```
Comment (brylie) [inline README.md]: Perhaps we can distill this article to the vital steps, and include them in the README?
```suggestion
[Docker](https://www.docker.com/) makes it easy to [install and manage Postgres](https://hackernoon.com/dont-install-postgres-docker-pull-postgres-bee20e200198).

With Docker installed, try the following steps:

1. `docker pull postgres`
2. `mkdir -p $HOME/docker/volumes/postgres`
3. docker run --rm   --name pg-docker -e POSTGRES_PASSWORD=docker -d -p 5432:5432 -v $HOME/docker/volumes/postgres:/var/lib/postgresql/data  postgres
```

Postgres should then be accessible via localhost, e.g. from Django or with the `psql` command line tool:

```
psql -h localhost -U postgres -d postgres
```

**Comment by BethanyG** [comment]
With the new settings file using `django-environ`, django will expect an `env variable` (either in the environment or in a `.env` file) called **DATABASE_URL**.  It should take the form of this:

postgres://{**user**}:{**password**}@{**hostname**}:{**port**}/{**database-name**}

**as an example:**
postgres://silly_username:a_password@localhost:5342/your_db_name_here

`env.db()` will then parse this into a DATABASE dict automatically for Djangos use in settings.

File: README.md (hunk 1 above)
Comment (lpatmo) [reply README.md]: Thank you! And sorry, I was being a bit lazy/doing a rush job of this PR last night. :P

**Comment by brylie** [comment]
If there are no other improvements, let's go ahead and merge this PR :-)

**Comment by lpatmo** [comment]
Oof... I know these instructions I wrote up quickly aren't the best and @bengineerdavis wanted to work on something clearer, but they are better than the status quo at the moment. So will merge so it's in history, and we can update these in a future PR.

**Pull request merged**